e2e/wdio/core/pages/split-button.po.ts

```diff
@@ -0,0 +1,46 @@
+import { CoreBaseComponentPo } from './core-base-component.po';
+import { waitForElDisplayed, waitForPresent, getElementArrayLength } from '../../driver/wdio';
+
+export class SplitButtonPo extends CoreBaseComponentPo {
+    private url = '/splitButton';
+
+    root = '#page-content';
+
+    buttonBehaviorExample = 'fd-split-button-behaviors-example ';
+    iconBehaviorExample = 'fd-split-button-icons-example';
+    buttonTypesExample = 'fd-split-button-types-example';
+    buttonPragmaticalExample = 'fd-split-button-programmatical-example';
+    buttonTemplateExample = 'fd-split-button-template-example';
```

Comment:
if you're going to use these like in my examples (selector: example + button), then you need to add a space at the end like on the 1st one.